plugins/package-managers/nuget/src/funTest/assets/dotnet-with-csproj-and-nuget-config-output.yml

```diff
@@ -23,17 +23,19 @@ packages:
 - id: "NuGet::xunit.analyzers:1.2.0-pre.7"
   purl: "pkg:nuget/xunit.analyzers@1.2.0-pre.7"
   authors:
-  - "jnewkirk, bradwilson, marcind"
-  declared_licenses: []
-  declared_licenses_processed: {}
+  - "jnewkirk,bradwilson,marcind"
```

Comment:
Is this commit related to other commits of this PR?

Reply:
Not strictly, see the [note above](https://github.com/oss-review-toolkit/ort/pull/6971#issuecomment-1539504175). The commit was created via the mechanism introduced in the PR, as a coincident showcase basically.